tests/search/test-class-search.php

```diff
@@ -2830,6 +2830,41 @@ public function test__maybe_enable_ep_query_logging_debug_bar_and_qm_enabled() {
 		$this->assertTrue( WP_EP_DEBUG );
 	}
 
+	public function test__are_es_constants_defined__no_constatns() {
+		$result = \Automattic\VIP\Search\Search::are_es_constants_defined();
+
+		$this->assertFalse( $result );
+	}
+
+	public function test__are_es_constants_defined__all_constatns() {
+		define( 'VIP_ELASTICSEARCH_ENDPOINTS', [] );
+		define( 'VIP_ELASTICSEARCH_USERNAME', 'foo' );
+		define( 'VIP_ELASTICSEARCH_PASSWORD', 'bar' );
+
+		$result = \Automattic\VIP\Search\Search::are_es_constants_defined();
+
+		$this->assertTrue( $result );
+	}
+
+	public function test__are_es_constants_defined__empty_password() {
+		define( 'VIP_ELASTICSEARCH_ENDPOINTS', [] );
+		define( 'VIP_ELASTICSEARCH_USERNAME', 'foo' );
+		define( 'VIP_ELASTICSEARCH_PASSWORD', '' );
+
+		$result = \Automattic\VIP\Search\Search::are_es_constants_defined();
+
+		$this->assertFalse( $result );
+	}
+
+	public function test__are_es_constants_defined__no_username() {
+		define( 'VIP_ELASTICSEARCH_ENDPOINTS', [] );
+		define( 'VIP_ELASTICSEARCH_PASSWORD', 'bar' );
+
+		$result = \Automattic\VIP\Search\Search::are_es_constants_defined();
+
+		$this->assertTrue( $result );
```

Comment:
```suggestion
		$this->assertFalse( $result );
```

Reply:
I think there's a typo here, that causes test to fail

Reply:
Perfect thanks!. Fixed.